Test errors with React 16 (probably needs enzyme-to-json update)

We pull in `enzyme-to-json@^1.4.4` via `jest-serializer-enzyme`.  But we probably need `enzyme-to-json@3.0.1` in order to work with React 16.

Blocks #34 #35 #36 #36 

Action plan: wait to see whether `jest-serializer-enzyme` is updated shortly.  It already has a [PR](https://github.com/rogeliog/jest-serializer-enzyme/pull/16) for this.

Errors look like
```
 FAIL  src/view/token.spec.js
  ● Test suite failed to run

    Cannot find module 'enzyme/build/ShallowTraversal' from 'shallow.js'
      
      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:191:17)
      at Object.<anonymous> (node_modules/enzyme-to-json/build/shallow.js:25:25)

```

--- a/package.json
+++ b/package.json
@@ -34,8 +34,8 @@
     "big.js": "^4.0.0",
     "copy-webpack-plugin": "^4.0.1",
     "enzyme": "^2.8.2",
+    "enzyme-to-json": "2",
     "jest": "^21.0.2",
-    "jest-serializer-enzyme": "^1.0.0",
     "prop-types": "^15.5.10",
     "react": "^15.5.4",
     "react-dom": "^15.5.4",
@@ -68,7 +68,7 @@
       }
     },
     "snapshotSerializers": [
-      "<rootDir>/node_modules/jest-serializer-enzyme"
+      "enzyme-to-json/serializer"
     ]
   }
 }

